Makefile

```diff
@@ -3,7 +3,7 @@ export GO111MODULE
 
 SERVER_EXE_EXT ?=
 DOCKER_CMD ?= docker
-DOCKER_REPO ?= docker.io/joaquim
+DOCKER_REPO ?= docker.io/kinvolk
```

Comment:
This was on my todo list :slightly_smiling_face: 
Can you please move it into a different commit though (let's keep things atomic)?

Reply:
This done.